src/explore-education-statistics-admin/src/pages/release/edit-release/manage-datablocks/components/ChartAxisConfiguration.tsx

```diff
@@ -239,6 +239,64 @@ const ChartAxisConfiguration = ({
               />
             )}
           </FormGroup>
+          <FormFieldset
+            id={`${id}-label`}
+            legend="Axis label"
+            legendSize="m"
+            hint="Label your axes"
+          >
+            <FormFieldTextInput
+              id={`${id}-label.value`}
+              label="Axis label"
+              name="label.value"
+              width={10}
+            />
+            <FormFieldTextInput
+              id={`${id}-label.angle`}
+              label="Axis label angle"
+              name="label.angle"
+              type="number"
+              min="-180"
+              max="180"
+              width={4}
+            />
+            <FormFieldSelect
+              id={`${id}-label.position`}
+              label="Axis label position"
+              name="label.position"
+              options={[
+                'top',
+                'bottom',
+                'center',
+                'end',
+                'left',
+                'right',
+                'inside',
+                'outside',
+                'insideLeft',
+                'insideRight',
+                'insideTop',
+                'insideBottom',
+                'insideTopLeft',
+                'insideBottomLeft',
+                'insideTopRight',
+                'insideBottomRight',
+                'insideStart',
+                'insideEnd',
+              ].map(position => {
+                return { label: noCase(position), value: position };
+              })}
+            />
+            <FormFieldTextInput
+              id={`${id}-label.offset`}
+              label="Axis label position offset"
+              name="label.offset"
```

Comment:
If we're letting the user change the offset of the label, shouldn't this tie in with the axis size as well?